mobify/scooch
# Hide arrow controls when at carousel bounds

Add a config option that will auto-hide arrow controls when at the carousel's bounds (Hide 'previous' control on first slide and hide 'next' control on last slide). Default is not to perform this behaviour.

**Status**: _Opened for visibility_

**Reviewers**: @tedtate @scalvert @kpeatt 
## Changes
- I've noticed this request from clients quite a bit and thought that we might consider adding this behaviour into the Scooch plugin
- Added a new config option for Scooch: `autoHideArrows`, default to false
- If enabled, hides/shows carousel control arrows in `afterSlide` event handler
## How to test-drive this PR
- Pull down this change branch
- Init a new Scooch with `autoHideArrows` option set to true: `$scooch.scooch({autoHideArrows: true});`
- Check whether arrow controls hide at first and last slides
